app/routes.py

```diff
@@ -1,2 +1,76 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, request
+from app.models.planet import Planet
+from app import db
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods = ["GET", "POST"])
+def get_planets():
```

Comment:
Consider separating each request method into its different functions. This makes testing the endpoints much easier to do.